Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Scripts/Input/Controllers/OculusHand.cs

```diff
@@ -244,77 +196,20 @@ public void UpdateController(OVRHand hand, OVRSkeleton ovrSkeleton, Transform tr
                         }
                         break;
                     case DeviceInputType.IndexFinger:
-                        UpdateIndexFingerData(Interactions[i]);
+                        Interactions[i].PoseData = currentIndexPose;
+                        if (Interactions[i].Changed)
+                        {
+                            CoreServices.InputSystem?.RaisePoseInputChanged(InputSource, ControllerHandedness, Interactions[i].MixedRealityInputAction, currentIndexPose);
+                        }
```

Comment:
Can this call into the definition instead?

```suggestion
                        handDefinition.UpdateCurrentIndexPose(Interactions[i]);
```

Reply:
I'm also not sure if `currentIndexPose` is being updated with the removed call to `UpdateIndexFingerData`